app/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -19,10 +19,10 @@
         android:icon="@drawable/ic_nav_receive"
         android:title="@string/Button.receive" />
 
```

Comment:
Important that the Tab text has "BUY / RECEIVE" or "BUY RECEIVE" more important than the image (which we will remove)